### PR TITLE
Break/Decode Game Info Request from Client GetCurrentGames

### PR DESCRIPTION
**Pull request by Adevill20**
As discussed in forum ( https://community.playfab.com/questions/14314/ue4-blueprint-decodegetmatchmakergameinforesult.html?childToView=14837#comment-14837 )

Also updated plugin file to reflect compatibility with 4.17 (also compatible with 4.18 thus far) as well as listing the dependency on the "OnlineSubsystemUtils" to remove the irritating message.

### CODE REVIEW

**Comment by pgilmorepf** [comment]
Heya, sorry for the delay on this.
We're going to try and get this into the generator, so that ALL lists generate in this way, correctly.
We're also trying to update to 4.18, and fix some unrelated minor issues.
It's all a work in progress, so stay tuned.

File: PlayFabSDK/PluginFiles/PlayFab/PlayFab.uplugin [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
+  "Modules": [
+    {
+      "Name": "PlayFab",
+      "Type": "Runtime",
+      "LoadingPhase": "PreDefault"
+    }
+  ],
+  "Plugins": [
+    {
+      "Name": "OnlineSubsystemUtils",
```
[the remark this answers is not in the document]
Comment (Adevill20) [reply PlayFabSDK/PluginFiles/PlayFab/PlayFab.uplugin]: The unreal build system stated that the "OnlineSubsystemUtils" plugin is missing, even though it is being used. Maybe a reference remained some where ?

[the remark this answers is not in the document]
Comment (Adevill20) [reply PlayFabSDK/PluginFiles/PlayFab/PlayFab.uplugin]: Thanks for all the effort. I'll give it a try and give you some feedback ASAP.

**Comment by pgilmorepf** [comment]
I'm going to close this issue. I know that this would be an improvement, but I ultimately want to take this in a wholly different direction, and make the blueprints just easier to use at a lower level.

That change is going to be incompatible with this one, but I don't have time to allocate to Unreal right now.  So, for now, feel free to make the changes you need on your own, and someday we'll implement the improved version.

**Comment by Adevill20** [comment]
@pgilmorepf  any update on the progress of the new direction you wanted to take ?

**Comment by pgilmorepf** [comment]
I ultimately want to replace essentially all of the arbitrary-json sub-objects with structured objects that can be broken much like the initial result objects are now.

I started on that, and it snowballed quickly into a huge amount of work. I had to put it on hold due to other PlayFab work in the pipeline.

Basically, because this is built with the generator, it's all-or-nothing. I can't just convert one result-sub-object for one API.

**Pull request closed without merging**